tsconfig.json

```diff
@@ -8,12 +8,12 @@
     "strict": true,
     "declaration": true,
     "forceConsistentCasingInFileNames": true,
-    "importHelpers": true,
```

Comment:
This was related to tslib dep - keeps polyfills, and was used to be used to target ES5.